pkg/apis/alerterator/v1alpha1/types.go

```diff
@@ -59,7 +67,7 @@ type Rule struct {
 
 type AlertSpec struct {
 	Receivers Receivers `json:"receivers"`
-	Alerts    []Rule    `json:"alerts"`
+	Alerts    []Rule    `json:"alerts" validate:"dive"`
```

Comment:
Hva betyr dette?

Reply:
Validatoren validerer ikke hvert element i arrayen med mindre man spesifiserer `dive` (se https://godoc.org/gopkg.in/go-playground/validator.v9#hdr-Dive)